kansa/lib/upgrade.js

```diff
@@ -71,15 +65,15 @@ function upgradeMembership(req, db, data) {
             WHERE id=$(id)
         RETURNING email, member_number`, data);
 
-    case 2:
+    case 3:
       email = prev.email;
       member_number = prev.member_number;
       const log = new LogEntry(req, `Upgrade to ${data.membership}`);
       if (data.paper_pubs) log.description += ' and add paper pubs';
       log.subject = data.id;
       return tx.none(`INSERT INTO Log ${log.sqlValues}`, log);
 
-    case 3:
+    case 4:
```

Comment:
More magic numbers, could we name these or leave a comment about them?
What happens when `i` doensn't fit between 0 and 4?